common/solar_system_initializers/canon/06_canon_human_systems_alliance_initializer.txt

```diff
@@ -140,10 +141,11 @@ sys_sol_init = {
 			generate_btr_start_deposits_and_blockers = yes
 			generate_hum_earth_capitol_buildings = yes
 			generate_hum_earth_pops = yes
-			add_deposit = d_vancouver
```

Comment:
Why'd we remove the custom features we already have? Are we doing the same for Thessia and the MEC features? I was under the impression that we'd leave them since they were within lore and helped with distinguishing the locations.